source/js/buyers-guide/components/primary-nav/primary-nav.scss

```diff
@@ -1,3 +1,5 @@
+$pni-primary-nav-breakpoint-larger: $bp-sm;
```

Comment:
it feels odd to see "breakpoint-larger" but then have a value "sm" (a bit like seeing `$bold: 100` =), can we think of a better name for this breakpoint?

Reply:
I know but this is tricky. Maybe we can just call it `$pni-primary-nav-breakpoint`